csv data source/dockerfile

```diff
@@ -1,10 +1,24 @@
-FROM python:3.11.1-slim-buster
+# Dependency-building stage.
+FROM python:3.11.1-slim-buster AS build
 
 ENV DEBIAN_FRONTEND="noninteractive"
 ENV PYTHONUNBUFFERED=1
 ENV PYTHONIOENCODING=UTF-8
 
 WORKDIR /app
+RUN python3 -m venv env
+ENV PATH="env/bin:$PATH"
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt --extra-index-url https://pkgs.dev.azure.com/quix-analytics/53f7fe95-59fe-4307-b479-2473b96de6d1/_packaging/public/pypi/simple/
```

Comment:
Can you update all the pip installs too please..

should be just normal:

`RUN python3 -m pip install -r requirements.txt`

Reply:
Oh nice. Thanks @SteveRosam. I've updated the PR. :-)